integration_tests/juniper_tests/src/issue_922.rs

```diff
@@ -0,0 +1,121 @@
+use juniper::*;
+
+struct Query;
+
+#[juniper::graphql_object]
+impl Query {
+    fn characters(executor: &Executor) -> Vec<CharacterValue> {
+        executor.look_ahead();
```

Comment:
@ArsileLuci why do we need `executor` here? Does it influence test somehow? Remove it, if it's unnecessary.